### ISSUE TITLE
Dragging cards is a struggle

### ISSUE THREAD

**Issue by Gozala** [inkandswitch/pushpin#341]
I keep struggling with moving text cards on the board. Because there is a tiny space to where they can be grabbed. I feel like some frame around the cards might go against design goals, but I can't help but think that it might improve overall user experience by creating intuition & visual space for holding cards.

**Comment by pvh** [comment]
Yes, the current drag implementation is buggy. I think it's less of a model problem and more of an implementation one -- can you think of an application you use that does this well? I absolutely want to spend some cycles on this. I think several applications do a good job -- Google Sheets, Trello, Milanote, Google Keep, Finder -- but I think the one that most closely models my expectations is Milanote. Do you have an app that does better I should compare to?
